docs/src/appendix/cast/multicall.md

```diff
@@ -27,4 +34,19 @@ call_type = "invoke"
 contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
-```
\ No newline at end of file
+```
+
+## `new`
+
+Generates an empty template for the multicall `.toml` file that may be later use with the `run` subcommand. It either outputs it to a new file or to the standard output.
+
+### `--output_path, -p <PATH>`
+Optional.
+
+When provided, it specifies where the template should be saved, it has to be a file path.
+If omitted, the template contents are going to be printed out to the stdout.
```

Comment:
```suggestion
Specifies a file path where the template should be saved.
If omitted, the template contents will be printed out to the stdout.
```

Reply:
ok what's wrong with 

> When provided, it specifies

?
As for 

> it has to be a file path

you yourself said we should mention this explicitly [here](https://github.com/foundry-rs/starknet-foundry/pull/98#discussion_r1268026976).

Reply:
[are going to => will](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/70be9ff5d0ce5e0b497e0cc9d686e59f0df9dacd)

Reply:
> The diff I propose states that explicitly - `Specifies a file path` - and not `Specifies a path`.

ok fair I missed that

Reply:
[updt](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/222bb30e6236945f904fd79a0cc66021efc637f2)